eslint.config.mjs

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+import eslint from "@eslint/js";
+import tseslint from "typescript-eslint";
+import pluginReact from "eslint-plugin-react";
+import eslintPluginPrettierRecommended from "eslint-plugin-prettier/recommended";
+
+export default [
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  pluginReact.configs.flat?.recommended,
```

Comment:
I don't like the optional chain here -- I suspect we'll just want a ts-expect-error like we have in https://github.com/alveusgg/alveusgg/blob/main/apps/website/eslint.config.js?